src/init.cpp

```diff
@@ -393,7 +395,7 @@ void SetupServerArgs(NodeContext& node)
 #if HAVE_SYSTEM
     argsman.AddArg("-alertnotify=<cmd>", "Execute command when a relevant alert is received or we see a really long fork (%s in cmd is replaced by message)", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
 #endif
-    argsman.AddArg("-assumevalid=<hex>", strprintf("If this block is in the chain assume that it and its ancestors are valid and potentially skip their script verification (0 to verify all, default: %s, testnet: %s)", defaultChainParams->GetConsensus().defaultAssumeValid.GetHex(), testnetChainParams->GetConsensus().defaultAssumeValid.GetHex()), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-assumevalid=<hex>", strprintf("If this block is in the chain assume that it and its ancestors are valid and potentially skip their script verification (0 to verify all, default: %s, testnet: %s, signet: %s)", defaultChainParams->GetConsensus().defaultAssumeValid.GetHex(), testnetChainParams->GetConsensus().defaultAssumeValid.GetHex(), signetChainParams->GetConsensus().defaultAssumeValid.GetHex()), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
```text
  -assumevalid=<hex>
       If this block is in the chain assume that it and its ancestors are valid
       and potentially skip their script verification (0 to verify all,
       default:
       0000000000000000000f2adce67e49b0b6bdeb9de8b7c3d7e93b21e7fc1e819d,
       testnet:
       000000000000056c49030c174179b52a928c870e6e8a822c75973b7970cfbd01,
       signet:
       0000000000000000000000000000000000000000000000000000000000000000)

  -minimumchainwork=<hex>
       Minimum work assumed to exist on a valid chain in hex (default:
       00000000000000000000000000000000000000000e1ab5ec9348e9f4b8eb8154,
       testnet:
       0000000000000000000000000000000000000000000001495c1d5a01e2af8a23,
       signet:
       0000000000000000000000000000000000000000000000000000000000000000)
```
Are the values for signet correct?

Reply:
Per `src/chainparams.cpp`, apparently so.

Reply:
@kallewoof ?

Reply:
Yes, signet has no assume valid chain, and since there can be custom signets, we don't wanna assume anything. We may put the genesis block hash there now that it's permanent across all signets, though, but 000 is probably equally fine.